client/finality-grandpa/src/communication/mod.rs

```diff
@@ -272,10 +272,11 @@ impl<B: BlockT, N: Network<B>> NetworkBridge<B, N> {
 	/// network all within the current set.
 	pub(crate) fn round_communication(
 		&self,
+		keystore: &Option<BareCryptoStorePtr>,
```

Comment:
Maybe `Option<&BareCryptoStorePtr>` instead?

Reply:
You're right that the `Option` doesn't need to be a reference. But `BareCryptoStorePtr` is basically an Arc so i just removed the reference`.